src/main/java/com/feedzai/commons/sql/abstraction/dml/Concat.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright 2020 Feedzai
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.feedzai.commons.sql.abstraction.dml;
+
+import java.util.Collection;
+
+/**
+ * PDB Concat Operator.
+ *
+ * @author Francisco Santos (francisco.santos@feedzai.com)
+ * @since @@@feedzai.next.release@@@
```

Comment:
I'm not sure that PDB has the materialization for these tokens or if you should just hard code the version.

Reply:
Yeah I don't let's make it `2.8.1`